### PR TITLE
[EG] add SysEvents beta package

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

File: sdk/eventgrid/azure-eventgrid-system-events/tests/test.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import pytest
+from azure.eventgrid.system.events.models import AcsChatEventBaseProperties
+from devtools_testutils import AzureRecordedTestCase
+
+
+class SystemEventTest(AzureRecordedTestCase):
+
+    @pytest.mark.live_test_only
```
Comment (l0lawrence) [inline sdk/eventgrid/azure-eventgrid-system-events/tests/test.py:8]:
```suggestion
```

File: sdk/eventgrid/azure-eventgrid-system-events/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
+
+# Azure Event Grid System Events client library for Python
+<!-- write necessary description of service -->
```
Comment (kashifkhan) [inline sdk/eventgrid/azure-eventgrid-system-events/README.md:4]: any sort of description/info or links we want to add here?

File: sdk/eventgrid/azure-eventgrid-system-events/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+#### Prequisites
+
+- Python 3.8 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Azure Event Grid System Events instance.
+
+## Key concepts
+## Examples
+## Troubleshooting
+## Next steps
```
Comment (kashifkhan) [inline sdk/eventgrid/azure-eventgrid-system-events/README.md:23]: if these sections arent going to be filled lets remove them for now :)

Comment (l0lawrence) [reply sdk/eventgrid/azure-eventgrid-system-events/README.md:23]: I didnt have them earlier but the readme check wants them

**Review by kashifkhan — APPROVED** [review]

**Pull request closed without merging**